### DISCUSSION TITLE
Welcome to pythonbible Discussions!

### DISCUSSION THREAD

**Discussion by avendesora** [avendesora/pythonbible#67] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by FlatrockITSolutions** [comment]
Are there more versions coming out? I would like to use NKJV and AMP in my discord bot: https://github.com/FlatrockDesigns/The-Kevin-Project
I am happy to help if I know how to format the data.
